Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift

```diff
@@ -1213,56 +1213,6 @@ final class LinkGeneratorTests: XCTestCase {
         ])
     }
 
-    func test_generateCopyExecutablesBuildPhase() throws {
```

Comment:
What about adding a test [with this scenario](https://github.com/tuist/tuist/issues/7063) to prevent regressions from being introduced in this area?